web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx

```diff
@@ -85,18 +120,22 @@ const FeedbackTemplateSelector = (props) => {
       />
       }
       <div className="card-container">
-        {getTemplates().map((template) => (
-          <TemplateCard
-            key={template.id}
-            title={template.title}
-            creator={template.creator}
-            description={template.description}
-            isAdHoc={template.isAdHoc}
-            questions={template.questions}
-            expanded={preview.open}
-            onClick={(e) => handlePreviewOpen(e, template)}
-            onCardClick={() => onCardClick(template)}/>
-        ))}
+        {
+          (filteredTemplates.length === 0 && searchText === "")
+              ? <h2>No templates found</h2>
+              : (searchText && searchText.length >= 0 && filteredTemplates.length === 0)
```

Comment:
This can also use the Truthy/Falsy evaluation to write it like so -> (searchText && !filteredTemplates.length)